src/journal/journald-audit.c

```diff
@@ -269,6 +264,13 @@ static int map_all_fields(
                 const MapField *mf;
                 const char *v;
 
+                if (*n == m) {
```

Comment:
even though it's equvialent, i think it's more explanatory to do a `>=` check here. i.e. it' think it's nicer to declare the "invalid range" here, instead of just the border between the valid and invalid range, if you follow my babbling.... ;-)